src/autowrap.lisp

```diff
@@ -8,6 +8,7 @@
                     "/usr/include/"
                     "/usr/include/arm-linux-gnueabihf"
                     "/usr/local/include/SDL2")
+  :sysincludes `,(uiop:split-string (uiop:getenv "C_INCLUDE_PATH") :separator ":")
```

Comment:
Hmm, OK, but can you make the inclusion of `:sysincludes` conditional on `C_INCLUDE_PATH` being set?  I do not know what `:sysincludes nil` does to other platforms.  Also, can you document how to use this, even just in a comment?  Does `nix shell` automatically export `C_INCLUDE_PATH` or is the idea the user has to set it prior to doing something?

Reply:
I could not find a way to set :SYSINCLUDES conditionally to the presence of the `C_INCLUDE_PATH` environment variable; however, I did take a look at the C-INCLUDE macro-expansion, and `sysincludes` just gets EVAL-d, and since it's initialized with NIL, it should not make a difference if we pass an empty list or omit it altogether.

Also, to address your question about `C_INCLUDE_PATH`, I went on and added a new section to the README explaining how to regenerate bindings and how `C_INCLUDE_PATH` can be used if the default configuration was not working.